bigquery/indexer.py

```diff
@@ -16,16 +16,38 @@
     datefmt='%Y%m%d%H:%M:%S')
 logger = logging.getLogger('indexer.bigquery')
 
+UPDATE_SAMPLES_SCRIPT = """
+if (!ctx._source.containsKey('samples')) {
+   ctx._source.samples = [params.sample]
+} else {
+   // If this sample already exists, merge it with the new one.
```

Comment:
Note: this extra logic around merging wouldn't be needed if we just blow away the index before starting. However, I think that's not a great approach if this is run as a cron job because it will result in down time for the data explorer.